index.html

```diff
@@ -51,18 +51,154 @@ <h1 class="big-title">
           </div>
         </div>
         <div class="part-two">
-          <img src="big-meal.png" alt="big meal" />
+          <img src="big-meal.png" alt="big meal" class='big-meal-photo' />
         </div>
       </section>
-      <section>
-        <h2>Second section</h2>
-        <div>Add text and images</div>
+      <section class="section-two">
+          <div class="head-meal-section">
+            <h4 class="head-meal-title">
+              Don't just pick a restaurant </br>
+              <span class="red-span">
+                pick your meal
+              </span>
+            </h4>
+          </div>
+          <p class="under-head-meal-section">
+            Gustar offers you the perfect meal options based on your<br />
+            favorite food, places to eat, and budget.
+          </p>
+          <div class="about-you">
+            <div class="vege-card">
+              <img src="carrot.png" width = "50em" height= "50em"  />
```

Comment:
lint

Reply:
I use linter and on only img element it set with space on the end like this.

Reply:
**width = "50em"** This is the problem, I saw it throughout the PR. Spacing at the end is fine except it adds two spaces which is not necessary.